styx-api-service/src/main/java/com/spotify/styx/api/WorkflowResource.java

```diff
@@ -232,10 +220,6 @@ public WorkflowResource(Storage storage, String schedulerServiceBaseUrl) {
     }
   }
 
-  private String schedulerApiUrl(CharSequence... parts) {
```

Comment:
Is this strictly related to this PR or rather a rest from the prior POST api?

Reply:
Yeah, it is a clean up.